templates/stats/publications.tpl

```diff
@@ -184,30 +184,33 @@
 					<pkp-table
 						labelled-by="publicationDetailTableLabel"
 						:class="tableClasses"
-						:columns="tableColumns"
-						:rows="items"
-						:order-by="orderBy"
-						:order-direction="orderDirection"
-						@order-by="setOrderBy"
+						@sort="setOrderBy"
 					>
```

Comment:
Have you tested that the ordering works? I don't have any stats generated so can't test it easily..  and thats something that might work quite differently to before.. so best to check that.

Reply:
Hi Jarda, yes the sorting still works. You might get some data in your local if you update the date range.